packages/macro/test/jsx-plural.ts

```diff
@@ -89,6 +89,42 @@ const cases: TestCase[] = [
         }} />;
       `,
   },
+  {
+    input: `
+        import { Trans, Plural} from '@lingui/macro';
+        <Plural
```

Comment:
could you also write a name for the test case, otherwise it's not clear what exactly we test there.